src/main/java/ch/bfh/bti7081/s2018/green/views/NavigationButton.java

```diff
@@ -30,7 +30,6 @@ public NavigationButton(String caption, AbstractComponentContainer containingVie
 
     private void changeStyle() {
         this.removeStyleNames("menuBarButton",".v-button", ".v-widget");
-        this.setId("navButton");
```

Comment:
change to this.addStyleName("navButton"); and #navButton to .navButton in files mytheme.scss and styles.css.
Or if this style isn't correct delete the styles in mytheme.scss and styles.css.

Reply:
So please delete the unneeded style with the '#navButton' in the files mytheme.scss and styles.css.

Reply:
this will be done in the styles pull request

Reply:
Already doing this - thx for remember me.
Our goal should be, to make the navigation menu not looking like a set of buttons - agree?